bridged-nft-token/src/mint.rs

```diff
@@ -0,0 +1,28 @@
+use crate::*;
+
+#[near_bindgen]
+impl Contract {
+    #[payable]
+    pub fn nft_mint(&mut self, token_id: TokenId, recipient: AccountId, metadata: TokenMetadata) {
+        let initial_storage_usage = env::storage_usage();
+        self.assert_owner();
+
+        let token = Token {
+            owner_id: recipient,
+            approved_account_ids: Default::default(),
+            next_approval_id: 0,
+        };
+        assert!(
+            self.tokens_by_id.insert(&token_id, &token).is_none(),
+            "Token already exists"
+        );
+        self.token_metadata_by_id.insert(&token_id, &metadata);
+        self.internal_add_token_to_owner(&token.owner_id, &token_id);
+
+        let new_token_size_in_bytes = env::storage_usage() - initial_storage_usage;
+        let required_storage_in_bytes =
+            self.extra_storage_in_bytes_per_token + new_token_size_in_bytes;
```

Comment:
This means that everyone is paying the base storage fee, though it only needs to be payed once.

The other option is that the first user to register a new token will pay a higher storage fee, but following users don't need to pay that fee.

While the second is not ideal, I think it is better than the current option.

```suggestion
        let required_storage_in_bytes = new_token_size_in_bytes;
```

Reply:
it was removed and replaced by the one contained inside the standard